cmdstanpy/model.py

```diff
@@ -486,7 +486,7 @@ def sample(
                     msg = '{}, chain {} returned error code {}'.format(
                         msg, i, stanfit._retcode(i)
                     )
-            raise Exception(msg)
+            print(msg)
```

Comment:
I don't think the csv files will parse if there's a non-zero ret code, so this is probably OK.